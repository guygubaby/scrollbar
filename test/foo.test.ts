import { expect, test } from 'vitest'

test('foo', () => {
  expect('foo').toEqual('foo')
  expect('foo').toMatchSnapshot()
})
