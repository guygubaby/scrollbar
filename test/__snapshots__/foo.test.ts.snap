// Vitest Snapshot v1

exports[`foo 1`] = `"foo"`;

exports[`test createScrollbarStyle 1`] = `
{
  "name": "custom",
  "scrollbarHeight": "8px",
  "scrollbarThumbColor": "#ddd",
  "scrollbarThumbRadius": "4px",
  "scrollbarTrackColor": "#f5f5f5",
  "scrollbarTrackRadius": "4px",
  "scrollbarWidth": "8px",
  "varPrefix": "",
}
`;

exports[`test createScrollbarStyle 2`] = `
"
  /* generated by scrollbar util */
  
  [scrollbar~=\\"custom\\"] {
    
  --scrollbar-width: 8px;
  --scrollbar-height: 8px;
  --scrollbar-track-radius: 4px;
  --scrollbar-thumb-radius: 4px;
  --scrollbar-track-color: #f5f5f5;
  --scrollbar-thumb-color: #ddd;
  
  }
  

  ::-webkit-scrollbar {
    width: var(--scrollbar-width);
    height: var(--scrollbar-height);
  }

  ::-webkit-scrollbar-track {
    background: var(--scrollbar-track-color);
    border-radius: var(--scrollbar-track-radius, 0);
  }

  ::-webkit-scrollbar-thumb {
    background: var(--scrollbar-thumb-color);
    border-radius: var(--scrollbar-thumb-radius, 0);
  }
"
`;

exports[`test generateVariables 1`] = `
"
  [scrollbar~=\\"custom\\"] {
    
  --scrollbar-width: 8px;
  --scrollbar-height: 8px;
  --scrollbar-track-radius: 4px;
  --scrollbar-thumb-radius: 4px;
  --scrollbar-track-color: #f5f5f5;
  --scrollbar-thumb-color: #ddd;
  
  }
  "
`;

exports[`test generateVariables 2`] = `
"
  :root {
    
  --scrollbar-width: 8px;
  --scrollbar-height: 8px;
  --scrollbar-track-radius: 4px;
  --scrollbar-thumb-radius: 4px;
  --scrollbar-track-color: #f5f5f5;
  --scrollbar-thumb-color: #ddd;
  
  }
  "
`;
